### ISSUE TITLE
Scraper: Scrape Issue Threads with Time estimates.

### ISSUE THREAD

**Issue by koya0** [koya0/.ubiquity-os#41]
Build a scraper that creates a JSONL compatible dataset, following OpenAI's spec for the fine-tune dataset, to be used with their framework. We need close to 250 to 300 well annotated and clean examples, for the training set. And a separate validation set which should have another 100 to 150 examples.

**Comment by koya0** [comment]
Build a scraper that creates a JSONL compatible dataset, following OpenAI's spec for the fine-tune dataset, to be used with their framework. [^01^]                     We need close to 250 to 300 well annotated and clean examples, for the training set. [^03^]                                                And a separate validation set which should have another 100 to 150 examples. [^04^] [^02^]                                




























































































































[^01^]: 87% similar to issue: [Scraper: Scrape Issue Threads with Time estimates.](https://www.github.com/koya0/.ubiquity-os/issues/41#41)

[^02^]: 65% similar to comment: [#22 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/22#issuecomment-2599399079)

[^03^]: 66% similar to comment: [#22 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/22#issuecomment-2599413405)

[^04^]: 67% similar to comment: [#22 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/22#issuecomment-2599393127)

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotaet https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2666933452 global

**Comment by koya0** [comment]
> 1. Estimates

Think that suggestions can be made from similar issue deadlines and time estimate labels but it is highly dependent on the specific repository's code base and team members' talent

There is a GitHub issue up to predict time labels. We can make another in the future for predicting deadlines which is dependent on this v1 to be up.

> 2. Incentives

I agree with your point perhaps we can start with 200% and then 100% at deadline and 0% at the end of the score decay timer.   

> `dual-timer`

Agreed but I think that it should be in v2 of the plugin. Basically we can make a task after v1 is up.

> In my previous career, I experienced an efficiency-based bonus system (1.5x – 3x hourly rates per hour of efficiency) that was highly effective. While salaries prevented base-rate reductions

Can you elaborate on how efficiency was measured?

Can you reiterate your last sentence? We use the term "base pay" as a parallel to salary so I am confused.  Less commonly we use the term "base rate" as a baseline multiplier for incentives in `text-conversation-rewards`.

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2676495258 global

**Comment by koya0** [comment]
> 1. Estimates

Think that suggestions can be made from similar issue deadlines and time estimate labels but it is highly dependent on the specific repository's code base and team members' talent

There is a GitHub issue up to predict time labels. We can make another in the future for predicting deadlines which is dependent on this v1 to be up.

> 2. Incentives

I agree with your point perhaps we can start with 200% and then 100% at deadline and 0% at the end of the score decay timer.   

> `dual-timer`

Agreed but I think that it should be in v2 of the plugin. Basically we can make a task after v1 is up.

> In my previous career, I experienced an efficiency-based bonus system (1.5x – 3x hourly rates per hour of efficiency) that was highly effective. While salaries prevented base-rate reductions

Can you elaborate on how efficiency was measured?

Can you reiterate your last sentence? We use the term "base pay" as a parallel to salary so I am confused.  Less commonly we use the term "base rate" as a baseline multiplier for incentives in `text-conversation-rewards`.

**Comment by koya0** [comment]
> 1. [^07^] [^06^] [^05^]  Estimates

Think that suggestions can be made from similar issue deadlines and time estimate labels but it is highly dependent on the specific repository's code base and team members' talent

There is a GitHub issue up to predict time labels. We can make another in the future for predicting deadlines which is dependent on this v1 to be up.

> 2. Incentives

I agree with your point perhaps we can start with 200% and then 100% at deadline and 0% at the end of the score decay timer. [^03^] [^02^]       

> `dual-timer`

Agreed but I think that it should be in v2 of the plugin. Basically we can make a task after v1 is up

> In my previous career, I experienced an efficiency-based bonus system (1.5x – 3x hourly rates per hour of efficiency) that was highly effective. While salaries prevented base-rate reductions

Can you elaborate on how efficiency was measured?

Can you reiterate your last sentence? We use the term "base pay" as a parallel to salary so I am confused. [^04^]      Less commonly we use the term "base rate" as a baseline multiplier for incentives in `text-conversation-rewards`. [^01^]  



















[^01^]: 73% similar to issue: [Automatic Priority and Time Labeling](https://www.github.com/koya0/.ubiquity-os/issues/33#33)

[^02^]: 78% similar to issue: [Deadlines Incentive](https://www.github.com/koya0/.ubiquity-os/issues/32#32)

[^03^]: 77% similar to comment: [#32 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/32#issuecomment-2637814124)

[^04^]: 78% similar to comment: [#32 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/32#issuecomment-2637851580)

[^05^]: 99% similar to comment: [#40 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/40#issuecomment-2680403083)

[^06^]: 99% similar to comment: [#33 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/33#issuecomment-2637958037)

[^07^]: 99% similar to comment: [#38 (comment)](https://www.github.com/koya0/.ubiquity-os/issues/38#issuecomment-2644205711)

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2676556916 global

**Comment by koya0** [comment]
/annotate https://github.com/koya0/.ubiquity-os/issues/41#issuecomment-2676556916 global
